source/administration/upgrade.rst

```diff
@@ -0,0 +1,232 @@
+
+3. Changes were made in 3.8.0 that require a change in the proxy configuration. If you're using NGINX:
+  1. Open the NGINX configuration file as root. The file is usually ``/etc/nginx/sites-available/mattermost`` but might be different on your system.
+  2. Locate the following line:
+     `location /api/v3/users/websocket {`
+  3. Replace the  line with ``location ~ /api/v[0-9]+/(users/)?websocket$ {``.
+
+  If you are using a proxy other than NGINX, make the equivalent change to that proxy's configuration.
+
+**To upgrade Mattermost Server**:
+
+1. In a terminal window on the server that hosts Mattermost Server, change to your home directory.
+
+  ``cd ~``
+
+2. Delete any files and directories that might still exist from a previous download.
+
+  .. code-block:: text
+
+    rm mattermost*.gz
+    rm -r mattermost
+
+3. Download `the latest version of the Mattermost Server <https://about.mattermost.com/download/>`_. In the following command, replace ``X.X.X`` with the version that you want to download:
+
+  Enterprise Edition
+    ``wget https://releases.mattermost.com/X.X.X/mattermost-X.X.X-linux-amd64.tar.gz``
+  Team Edition
+    ``wget https://releases.mattermost.com/X.X.X/mattermost-team-X.X.X-linux-amd64.tar.gz``
+
+4. Extract the Mattermost Server files.
+
+  ``tar -xzf mattermost*.gz``
+
+5. Make a copy of your configuration file. The existing file is overwritten during the upgrade, so it's important that you don't forget this step
+
+  ``cp {install-path}/mattermost/config/config.json config.json``
+
+6. Stop Mattermost Server.
+
+  On Ubuntu 14.04 and RHEL 6.6: ``sudo service mattermost stop``
+
+  On Ubuntu 16.04 and RHEL 7.1: ``sudo systemctl stop mattermost``
+
+7. Back up your data.
+  a. Back up your database using your organization’s standard procedures for backing up MySQL or PostgreSQL.
+  b. If you’re using local file storage, back up the location where files are stored.
+
+8. Copy the files that you extracted earlier to the install directory.
+
+  ``sudo cp -r mattermost {install-path}``
+
+9. Restore your configuration file.
+
+  ``sudo cp config.json {install-path}/mattermost/config``
+
+10. Change ownership of the new files.
+
+  ``sudo chown -R {owner}:{group} {install-path}/mattermost``
+
+11. Start Mattermost server.
+
+  On Ubuntu 14.04 and RHEL 6.6: ``sudo service mattermost start``
+
+  On Ubuntu 16.04 and RHEL 7.1: ``sudo systemctl start mattermost``
+
+12. If you have TLS set up on your Mattermost server, you must activate the CAP_NET_BIND_SERVICE capability to allow the new Mattermost binary to bind to low ports.
+
+  1. ``cd {install-path}``
+  2. ``sudo setcap cap_net_bind_service=+ep ./bin/platform``
+
+After the server is upgraded, users might need to refresh their browsers to experience any new features.
+
+Upgrading Team Edition to Enterprise Edition
```

Comment:
hm, good point.

I guess it's fine to keep, but it might be useful to add an FAQ for TE vs EE (differences in editions, and differences in downloads)